start.sh

```diff
@@ -2,4 +2,6 @@
 
 ./manage.py migrate
 
+# ./manage.py collectstatic
```

Comment:
Right now if this were uncommented deployments would fail with:

```
django.core.exceptions.ImproperlyConfigured: You're using the staticfiles app without having set the STATIC_ROOT setting to a filesystem path.
```

@duck-rh